src/classes/WavetableCache.tsx

```diff
@@ -0,0 +1,40 @@
+import Wavetable from "./Wavetable";
+import wavetables from "../data/wavetables";
+
+export default class WavetableCache {
```

Comment:
Is the purpose of the cache to persist wavetables and switch between them? As of right now it's only registering what lives in `wavetables.tsx`, but in the future we'd allow users to create their own, store them both in memory and persist it somehow, and then load it through here?